maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gitexe/src/test/java/org/apache/maven/scm/provider/git/gitexe/command/remoteinfo/GitExeRemoteInfoCommandTckTest.java

```diff
@@ -23,6 +23,8 @@
 import org.apache.maven.scm.provider.git.GitScmTestUtils;
 import org.apache.maven.scm.provider.git.command.remoteinfo.AbstractGitRemoteInfoCommandTckTest;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
I don't understand why this import is added; it is the only change in this file?

Reply:
In junit3 you inherited this method (w/ same signature from TestCase), but in Junit4 you need to use hamcrest, or Assert or alike, hence static import but no other change (as method sig is same).